website/content/configuration/troubleshooting.md

```diff
@@ -5,14 +5,17 @@ weight: 20
 
 Once you setup a loadbalancer on Kubernetes and it doesn't _seem_ to work right away, here are a few tips to troubleshoot.
 
-SSH into one or more of your nodes and use `arping` and `tcpdump` to verify the ARP requests pass through your network.
+- Check that the LoadBalancer service has endpoints (`kubectl get endpoints`) that are not pending - if they are, MetalLB will not respond to ARP requests for that service's external IP
```

Comment:
I think something common to do, also, is to check that the nodePort works as expected (to rule out other miscommunications, and it would catch also this). What do you think of adding that too?

Reply:
Ah, so manually checking that the service works?

Reply:
Yes, in my little experience on slack users channel, sometimes there is another unrelated issue that would have been catched if the service nodePort was tested (note all services type LoadBalancer are also NodePort) 

Do you feel comfortable to add a step to try that too? If not, I can do it.

Reply:
Hm, as I've not run into that I'm not sure what the best way to document it would be, sorry.

Reply:
No problem, created an issue to do this later: https://github.com/metallb/metallb/issues/576 then :)